src/actions.ts

```diff
@@ -13,6 +13,8 @@ export const ENABLE = 'ENABLE';
 export const DISABLE = 'DISABLE';
 export const ADD_RENDERER = 'ADD_RENDERER';
 export const REMOVE_RENDERER = 'REMOVE_RENDERER';
+export const ADD_INPUT = 'ADD_INPUT';
```

Comment:
Since we agreed on using the term 'Field', should we rename these occurrences (applies to reducers etc. as well)?